src/Components/Counter/__tests__/Counter.test.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { Counter } from '../Counter';
+
+describe('Counter tests', () => {
+  const initialValue = 5;
+
+  test('initial value is rendered correctly', async () => {
```

Comment:
It's customary to name test like this:

```suggestion
  test('renders initial value', async () => {
```

or 

```suggestion
  test('should render initial value', async () => {
```

These test names arguably are more clear. You can read them like "Counter renders initial value":

![image](https://github.com/user-attachments/assets/1877a1ab-31ae-4a3e-a7cf-62272614bccf)